geometric_map_server:
  ros__parameters:
    map_frame_id: map
